src/builder-main.c

```diff
@@ -156,6 +158,12 @@ static GOptionEntry show_deps_entries[] = {
   { NULL }
 };
 
+static GOptionEntry show_manifest_entries[] = {
+  { "verbose", 'v', 0, G_OPTION_ARG_NONE, &opt_verbose, "Print debug information during command processing", NULL },
+  { "show-manifest", 0, 0, G_OPTION_ARG_NONE, &opt_show_manifest, "Print out the manifest file in standard json format (see --show-manifest --help)", NULL },
```

Comment:
Shouldn't you remove `(see --show-manifest --help)` as this list will only appear when invoking that command? Same above too I guess.